3-end-of-the-world/src/main/scala/eotw/Interpreter.scala

```diff
@@ -0,0 +1,42 @@
+package io.github.sentenza.hacktoberfest.eotw
+
+import io.github.sentenza.hacktoberfest.fplibrary._
+import io.github.sentenza.hacktoberfest.application.Program
+
+object Interpreter {
+  def main(args: Array[String]): Unit = {
+    val description: IO[Unit]        = Program.createIO(args)
+    def interpret[A](desc: IO[A]): A = desc.unsafeRun()
+    print(Console.YELLOW)
```

Comment:
All the prints should be wrapped in IO, I guess

Reply:
I wanted to mark them as unsafe operations using `YELLOW`. It was on purpose.

Reply:
I didn't notice, fine!